### ISSUE TITLE
Enlarge icon size and improve alignment?

### ISSUE THREAD

**Issue by rfgamaral** [file-icons/vscode#14]
For me this is the best icon theme for VS Code but I find it distracting that the icons are tiny bit too small and don't feel properly aligned. Look at the following animation with a bunch of random icon themes for VS Code:

![file-icon-themes](https://cloud.githubusercontent.com/assets/96476/24898394/aee11e06-1e94-11e7-9b73-078d7c1842b5.gif)

Most of them have the same size (slightly larger than this one) and are better aligned between the item arrow and the item label.

Could this be improved in a future version or it's like this by design?

**Comment by DanBrooker** [comment]
I agree with you, but the reason the icons aren't aligned is because it's an icon font which can have variable size icons, in atom we have control over the css and can use text alignment, vscode seems to just always align the icons to the top left, rather than the centre. The other themes must have a fixed canvas size for icons and they've centred them inside that.

@Alhadis would be able to speak to this better. 

Changing all the icons likely be a lot of work, the only real solution would be to automate it somehow. Am open to suggestions.

**Comment by Alhadis** [comment]
@DanBrooker When you were bundling this package's icon-fonts, did you include the optimised versions I made ([DevOpicons](https://github.com/file-icons/DevOpicons)/[MFixx](https://github.com/file-icons/MFixx)), or the older, unmodified fonts? ([Devicons](https://github.com/vorillaz/devicons)/[MFizz](https://github.com/fizzed/font-mfizz))

The latter pair are the ones that're buggy and uneven. [They were replaced](https://github.com/file-icons/atom/releases/tag/v2.0.0#optimised-icons) in v2.0.0 with the ones I optimised manually (which was needed because there're no tools I know or trust that can perform automated sizing and alignment of icons).

Which is why I plan to develop one sooner or later.

**Comment by DanBrooker** [comment]
@Alhadis pretty sure I just grabbed the ones you're bundling into atom.

**Comment by Alhadis** [comment]
Ah, you did too. No idea then... and I'm not sure what VSCode could be doing, because I purposefully stripped margins from every icon. The only metric that differs between icons is advance width. Which might be a factor: File-Icons uses CSS for aligning icons horizontally within a box-sized region (which is more efficient/flexible than simply baking it into the icon itself).

But if VSCode doesn't use CSS for icons, there's not much we can do.

FYI, I wrote a [long, rambling explanation](https://github.com/file-icons/DevOpicons#size-fixes) on the topic of icon-margins. Anybody who's in a position to be sending it to an icon-font author who's doing their craft wrong is eagerly encouraged to do so.
